tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -0,0 +1,284 @@
+                SupportedFrameworks = new[] { TargetFramework.NETCOREAPP3_1, TargetFramework.NET5_0, TargetFramework.NET6_0 }
+            },
+            ExplorationTestName.swashbuckle => new ExplorationTestDescription()
+            {
+                GitRepositoryUrl = "https://github.com/domaindrivendev/Swashbuckle.AspNetCore.git",
+                Name = "Swashbuckle.AspNetCore",
+                GitRepositoryTag = "v6.2.3",
+                PathToUnitTestProject = "test/Swashbuckle.AspNetCore.SwaggerGen.Test",
+                SupportedFrameworks = new[] { TargetFramework.NET6_0 }
+            },
+            ExplorationTestName.paket => new ExplorationTestDescription()
+            {
+                GitRepositoryUrl = "https://github.com/fsprojects/Paket.git",
+                Name = "Paket",
+                GitRepositoryTag = "6.2.1",
+                PathToUnitTestProject = "tests/Paket.Tests",
+                TestsToIgnore = new[] { "Loading assembly metadata works" },
+                SupportedFrameworks = new[] { TargetFramework.NET461, TargetFramework.NETCOREAPP3_1 }
+            },
+            ExplorationTestName.ilspy => new ExplorationTestDescription()
+            {
+                GitRepositoryUrl = "https://github.com/icsharpcode/ILSpy.git",
+                Name = "ILSpy",
+                GitRepositoryTag = "v7.1",
+                PathToUnitTestProject = "ICSharpCode.Decompiler.Tests",
+                TestsToIgnore = new[] { "UseMc", "_net45", "ImplicitConversions", "ExplicitConversions", "ICSharpCode_Decompiler", "NewtonsoftJson_pcl_debug", "NRefactory_CSharp", "Random_TestCase_1", "AsyncForeach", "AsyncStreams", "AsyncUsing", "CS9_ExtensionGetEnumerator", "IndexRangeTest", "InterfaceTests", "UsingVariables" },
+                SupportedFrameworks = new[] { TargetFramework.NET461 },
+                IsTestedByVSTest = true
+            },
+            _ => throw new ArgumentOutOfRangeException(nameof(name), name, null)
+        };
+
+        return description;
+    }
+}
+
+partial class Build
+{
+    AbsolutePath ExplorationTestsDirectory => RootDirectory / "exploration-tests";
+
+    [Parameter("Indicates name of exploration test to run. If not specified, will run all tests sequentially.")]
+    readonly ExplorationTestName? ExplorationTestName;
+
+    [Parameter("Indicates whether exploration tests should skip repository cloning. Useful for local development. Default false.")]
+    readonly bool ExplorationTestSkipClone;
+
+    [Parameter("Indicates whether exploration tests should run on latest repository commit. Useful if you want to update tested repositories to the latest tags. Default false.", 
+               List = false)]
+    readonly bool ExplorationTestCloneLatest;
+
+    Target RunExplorationTests_Debugger
+        => _ => _
+               .Description("Run exploration tests for debugger.")
+               .After(Clean)
+               .DependsOn(BuildTracerHome)
```

Comment:
This shouldn't be necessary in the CI. You're already depending on the Windows build output, so this step has already been executed and you're duplicating work. I would either remove this `DependsOn()` here, or pass `--skip` in the `RunExplorationTests_*` step in the yaml

Reply:
Added `--skip` in the yml.
I want to keep this dependency for local development.

Reply:
> I want to keep this dependency for local development.

That's perfectly fine, though for weighing up the design of using `After`/`DependsOn` in targets, I consciously decided _against_ that approach for most targets. There's arguments both ways though, and if you've found that approach less error-prone/more convenient, then that's fine by me 🙂

Reply:
Replaced with `after`